src/20241210_AaveV3Ethereum_OrbitProgramRenewal/OrbitProgramRenewal.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Orbit Program Renewal - Q4 2024"
+author: "ACI"
+discussions: "https://governance.aave.com/t/arfc-orbit-program-renewal-q4-2024/20084"
+snapshot: "https://snapshot.box/#/s:aave.eth/proposal/0x60613deb2c662057cc8028b431df84fe6e763d38f48f70594a7cb7fd91a8cb93"
+---
+
+## Simple Summary
+
+Proposing the renewal of the Orbit program for recognized delegates, compensating them with GHO, associated with their governance activity during Q4 2024 ( From 2024-08-31 to 2024-12-14).
+
+## Motivation
+
+Orbit recognizes the added value of the Delegates in the decentralization & diversity of the Aave DAO. This compensation allows them to focus on Aave and keep their contribution efforts to our governance. The ACI proposes the extension of Orbit for a new quarter, Q4 2024, from 2024-08-31 to 2024-12-14.
+
+## Specification
+
+- **Period Coverage:** Q4 2024 from August 31th 2024 to December 14th 2024
+- **Eligible Platforms:**
+  - EzR3al
+  - Areta
+  - StableLab
+  - IgnasDefi
+- **Budget:** 60,000 GHO
+- **Relevant Links:**
+  - [ACI’s Orbit tracker](https://dapps.aavechan.com/orbit-tracker)
+
```

Comment:
i would also specify here how much is in streams, and how much in direct transfers (explanation more in line with proposal code)

Also would link the names of the platforms with the addresses in etherscan

Reply:
Forum post + PR markdown have been updated